Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -568,6 +580,30 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
       // specify the planes in view coordinates
       program.setUniform3f(`vPlaneNormal${i}`, normal[0], normal[1], normal[2]);
       program.setUniformf(`vPlaneDistance${i}`, dist);
+
+      if (actor.getProperty().getUseLabelOutline()) {
+        const image = model.currentInput;
+        const worldToIndex = image.getWorldToIndex();
+
+        program.setUniformMatrix('vWCtoIDX', worldToIndex);
+
+        // Compute the view coordinate to world coordinate transformation matrix.
+        model.viewToWorld = mat4.create();
+        mat4.invert(model.viewToWorld, keyMats.wcvc);
+
```

Comment:
model.viewToWorld seems to be created every frame and not used maybe?

Reply:
You're correct, I thought I caught of my dead playing around code. Nice catch, I'll remove it.